Library/Formula/userspace-rcu.rb

```diff
@@ -1,9 +1,7 @@
-require "formula"
-
 class UserspaceRcu < Formula
   homepage "http://lttng.org/urcu"
-  url "http://lttng.org/files/urcu/userspace-rcu-0.8.4.tar.bz2"
-  sha1 "4b3bf1b76e6ea50d9a56d9e5e00df2cc7c4d610f"
+  url "http://www.lttng.org/files/urcu/userspace-rcu-0.8.7.tar.bz2"
```

Comment:
Can you shunt both this and the `homepage` to `https` please? Upstream seem to support it.